### PR TITLE
Multilingual (Gujarati) Audio Classification using Wav2vec2.0 (Current State of the Art) using HuggingFace Framework

### PR DESCRIPTION
**Pull request by manthanthakker**
#### Final Accuracy on test set: 99.2

You can try the demo here: https://huggingface.co/manthan40/wav2vec2-base-finetuned-manthan_base
Steps to train the model: Simply run all cells in the notebook.

This PR includes:

1. A python notebook that loads a Wav2Vec2 model pretrained by Facebook through the hugging face framework. Finetunes on the Gujarati Dataset in the repository. 
2.  A new Hugging Face Dataloader and Tokenizer for Gujarati Dataset.

### CODE REVIEW

**Review by Nikunj1729 — COMMENTED** [review]
@manthanthakker PR looks good to me.

**Review by Nikunj1729 — APPROVED** [review]
